src/content/components/BugList/BugList.js

```diff
@@ -48,117 +62,190 @@ export class BugList extends React.PureComponent {
   }
 
   onCheck(e) {
-    const {checked, value} = e.target;
+    const { checked, value } = e.target;
     this.setState(prevState => {
       const newState = Object.assign({}, prevState.selectedBugs);
       if (checked) {
         newState[value] = true;
       } else {
         delete newState[value];
       }
-      return {selectedBugs: newState};
+      return { selectedBugs: newState };
     });
   }
 
   onCheckShowResolved(e) {
-    this.setState({showResolved: e.target.checked});
+    this.setState({ showResolved: e.target.checked });
   }
 
   getBulkEditLink(bugs) {
-    return `https://bugzilla.mozilla.org/buglist.cgi?bug_id=${bugs.join(",")}&order=bug_id&tweak=1`;
+    return `https://bugzilla.mozilla.org/buglist.cgi?bug_id=${bugs.join(
+      ","
+    )}&order=bug_id&tweak=1`;
   }
 
   renderFilters() {
-    return (<EditorGroup>
-      {this.props.showResolvedOption ? <span><input type="checkbox" onChange={this.onCheckShowResolved} checked={this.state.showResolved} /> Show Resolved</span> : null}
-    </EditorGroup>);
+    return (
+      <EditorGroup>
+        {this.props.showResolvedOption ? (
+          <span>
+            <input
+              type="checkbox"
+              onChange={this.onCheckShowResolved}
+              checked={this.state.showResolved}
+            />{" "}
+            Show Resolved
+          </span>
+        ) : null}
+      </EditorGroup>
+    );
   }
 
   renderBulkEdit(selectedBugs) {
-    return (<React.Fragment>
-      <EditorGroup>
-        <a className={gStyles.primaryButton} href={this.getBulkEditLink(selectedBugs)}>Edit in Bugzilla</a>
-      </EditorGroup>
-    </React.Fragment>);
+    return (
+      <React.Fragment>
+        <EditorGroup>
+          <a
+            className={gStyles.primaryButton}
+            href={this.getBulkEditLink(selectedBugs)}>
+            Edit in Bugzilla
+          </a>
+        </EditorGroup>
+      </React.Fragment>
+    );
   }
 
   filterResolved() {
-    const {bugs} = this.props;
-    if (this.state.showResolved) { return bugs; }
+    const { bugs } = this.props;
+    if (this.state.showResolved) {
+      return bugs;
+    }
     return bugs.filter(bug => !isBugResolvedOrMerged(bug));
   }
 
   _renderSubtitle() {
-    return (<span>
-      <strong>{this.props.meta ?
-        <a className={gStyles.plainLink} href={`https://bugzilla.mozilla.org/show_bug.cgi?id=${this.props.meta}`}>
-          {this.props.subtitle} - Bug {this.props.meta}
-        </a> :
-        this.props.subtitle}</strong> | </span>);
+    return (
+      <span>
+        <strong>
+          {this.props.meta ? (
+            <a
+              className={gStyles.plainLink}
+              href={`https://bugzilla.mozilla.org/show_bug.cgi?id=${
+                this.props.meta
+              }`}>
+              {this.props.subtitle} - Bug {this.props.meta}
+            </a>
+          ) : (
+            this.props.subtitle
+          )}
+        </strong>{" "}
+        |{" "}
```

Comment:
Is this ok?

Reply:
ah space, wow

Reply:
very strange O_o

Reply:
I wonder if we want to hack around prettier `</strong>{" | "}</span>` (or will prettier expand anyway…)